pkg/jsonlog/jsonlogbytes.go

```diff
@@ -23,8 +23,6 @@ func (mj *JSONLogBytes) MarshalJSONBuf(buf *bytes.Buffer) error {
 	if len(mj.Log) != 0 {
 		if first == true {
 			first = false
```

Comment:
Yeah you're right. I did that for ``jsonlog_marshalling.go`` here https://github.com/vdemeester/docker/blob/pkg-jsonlog-test-coverage/pkg/jsonlog/jsonlog_marshalling.go#L93 but didn't on this one :sweat_smile:.